metaflow/plugins/metadata/service.py

```diff
@@ -73,7 +88,55 @@ def register_task_id(self,
                            sys_tags=sys_tags)
         finally:
             self._register_code_package_metadata(run_id, step_name, task_id)
-        
+
+    def _start_heartbeat(self, heartbeat_type, flow_id, run_id, step_name=None, task_id=None):
+        if self._already_started():
+            raise Exception("heartbeat already started")
+        # start sidecar
+        if self.version() == 'local' or self.version() is None or \
```

Comment:
Do we expect the version to ever be 'local' here?

Reply:
i guess it never should go to this provide if running in local mode